### DISCUSSION TITLE
CSS Framework

### DISCUSSION THREAD

**Discussion by TFWIsaac** [JakyeRU/Larascord#26] [General]
Do you have any CSS Framework in your project? For example Tailwind or Bootstrap? I would be nice if this comes with your project. At the installation you could choose one of them for example.

**Comment by ValentinNeaga** [comment]
Hello @TFWIsaac,

Larascord uses [Laravel Breeze](https://laravel.com/docs/8.x/starter-kits#laravel-breeze) under the hood, which uses TailwindCSS. Unfortunately, it doesn't have the option to switch between TailwindCSS and Bootstrap.

**Comment by TFWIsaac** [comment]
Glad to here, cause i have the problem, that TailwindCSS does not work for me. I want to create a table with the css `table-auto` but i doesnt change anything. Also the [output of the app.css](https://pastebin.com/JX4vXR83) seems far to short for the hole TailwindCSS?

Do you have any suggestion?

**Reply by ValentinNeaga** [reply]
I'm not sure what could be wrong, but I suggest you try a table component from tailwindcomponents.com. [This one](https://tailwindcomponents.com/component/projects-table) seems promising.
